### PR TITLE
Import assets, Implement Navbar + Home page

### PR DESCRIPTION
**Pull request by vivz753**
Major changes:
- init pages (home, about-us, newsletter, programs)
- complete home page (excluding subscription form)
- complete navbar
- add assets

Assets include:
- fonts
- color palettes
- Vector SVGs

Screenshot
Screen Shot 2023-07-03 at 4 35 52 PM

Design reference (EP Declaration): https://www.figma.com/file/84nRBClWpH6JQsoa1HrKDN/Beta-Website-Design?type=design&node-id=12544%3A18466&mode=design&t=nVO6PnT9B7Ed0bF5-1

### CODE REVIEW

**Comment by jjasminew** [comment]
test

File: web/app/globals.css [hunk 1]
```diff
@@ -2,26 +2,30 @@
 @tailwind components;
 @tailwind utilities;
```
Comment (jjasminew) [inline web/app/globals.css:4]: test

File: web/app/globals.css [hunk 2]
```diff
@@ -2,26 +2,30 @@
...
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+
+@font-face {
+  font-family: "Poppins";
+	src: url("/fonts/Poppins/Poppins-Light.ttf") format("opentype");
+	font-weight: normal;
+	font-style: normal;
+}
```
Comment (jjasminew) [inline web/app/globals.css:11]: test2

File: web/app/globals.css (hunk 1 above)
Comment (vivz753) [reply web/app/globals.css:4]: thanks jasmine!

**Review by jjasminew — APPROVED** [review]

**Pull request merged**